src/theme/material/tooltip.m.css

```diff
@@ -1,2 +1,38 @@
+@import './variables.css';
+
+:root {
+	--mdc-tooltip-font-size: 0.975rem;
+}
+
 .root {
+	font-family: Roboto, sans-serif;
```

Comment:
can this be taken from an mdc var elsewhere?

Reply:
I didn't see anything... but did move this to `variables.css`